src/electionScraper.js

```diff
@@ -92,6 +93,62 @@ export class ElectionScraper {
         return electionLinks;
     }
 
+    /**
+     * @summary scrapes election page given the URL and returns an election map
```

Comment:
Why are we doing this? The elections index page will not show the historical elections when an active election is running.

Reply:
As I mentioned, options :) I haven't connected it anywhere, but if there are no active elections, a lot of information can be extracted from the page without actually hitting individual pages - I wanted to expand on the `Election` class a little bit later to accept a configuration object instead of only `electionURL` and election number to allow to get instances with some data already present. It will make more sense after everything's connected - sounds good?